src/cdk/keycodes/keycodes.ts

```diff
@@ -24,4 +24,6 @@ export const A = 65;
 export const Z = 90;
 export const ZERO = 48;
 export const NINE = 57;
+export const ZERO_NUMPAD = 96;
+export const NINE_NUMPAD = 105;
```

Comment:
I'd swap it to be `NUMPAD_ZERO`, `NUMPAD_NINE`

Reply:
Changed.